x-pack/plugins/watcher/server/routes/api/license/register_refresh_route.ts

```diff
@@ -13,11 +13,11 @@ it needs to make a round-trip to the kibana server. This refresh endpoint is pro
 for when the client needs to check the license, but doesn't need to pull data from the
 server for any reason, i.e., when adding a new watch.
 */
-export function registerRefreshRoute(deps: RouteDependencies) {
-  const handler: RequestHandler<any, any, any> = (ctx, request, response) => {
-    return response.ok({ body: { success: true } });
-  };
+export const handler: RequestHandler = (ctx, request, response) => {
```

Comment:
Same here, why do we export the handler?

Reply:
This was not necessary, I just thought it could be a nice pattern to follow if we want to write unit tests? But the unit tests would not include the validation logic of types, so perhaps not that useful.